examples/seismic/tutorials/06_elastic.ipynb

```diff
@@ -23,7 +23,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 11,
```

Comment:
Haven't looked at ti, but why is this notebook changing?

Reply:
Yeah, that's been annoying me. Will try and figure out why exactly.